lib/recipes-data/src/lib/transform.ts

```diff
@@ -2,30 +2,50 @@ import { FeaturedImageWidth, ImageDpr, PreviewImageWidth } from './config';
 import type { Contributor, RecipeImage } from './models';
 import { extractCropDataFromGuimUrl } from './utils';
 
-const getFastlyUrl = (
-	imageId: string,
-	cropId: string,
-	dpr: number,
-	desiredWidth: number,
-	originalWidth: number,
-  extension: string
-) => `https://i.guim.co.uk/img/media/${imageId}/${cropId}/master/${originalWidth}.${extension}?width=${desiredWidth}&dpr=${dpr}&s=none`;
+const getFastlyUrl = ({
+	imageId,
+	cropId,
+	dpr,
+	desiredWidth,
+	originalWidth,
+	extension,
+}: {
+	imageId: string;
+	cropId: string;
+	dpr: number;
+	desiredWidth: number;
+	originalWidth: number;
+	extension: string;
+}) =>
```

Comment:
It’s a shame this syntax is so verbose: it’s definitely nice to use an object for the calling code, but duplicating all the variable names here is a little annoying.

Reply:
Yes, it's a bit awkward! It's the only way to write something close to named parameters in JS/TS that I'm aware of – named params in Scala would I suppose be an example of a more concise syntax.